src/components/SingleList.jsx

```diff
@@ -11,8 +15,39 @@ export function SingleList({ name, path, setListPath, setLoading }) {
 		navigate('/list');
 	}
 
+	const handleDelete = async () => {
+		try {
+			if (!path.includes(user.uid)) {
+				if (
+					window.confirm(
+						`The ${name} list has been shared with you. Deleting it will remove it from your lists, and you won't be able to access it unless it's shared with you again. Are you sure you want to proceed with the deletion?`,
```

Comment:
Maybe a little more concise? `The ${name} list has been shared with you. Deleting it will remove it from your lists. You will loose access unless it's shared with you again. Are you sure you want to proceed with the deletion?`,